content/sensu-go/5.4/reference/checks.md

```diff
@@ -93,7 +72,219 @@ At every execution of a check command – regardless of success or failure – t
+This helps to balance the load of both the backend and the agent, and may result in a delay before initial check execution._
+
+**Example interval check**
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "interval_check",
+    "namespace": "default"
+  },
+  "spec": {
+    "command": "check-cpu.sh -w 75 -c 90",
+    "subscriptions": ["system"],
+    "handlers": ["slack"],
+    "interval": 60,
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+#### Cron scheduling
+
+You can also schedule checks using [cron syntax][14].
+For example, to schedule a check to execute once a minute at the start of the minute, set the `cron` attribute to `* * * * *` and the `publish` attribute to `true`.
+
+**Example cron check**
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "cron_check",
+    "namespace": "default"
+  },
+  "spec": {
+    "command": "check-cpu.sh -w 75 -c 90",
+    "subscriptions": ["system"],
+    "handlers": ["slack"],
+    "cron": "* * * * *",
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+#### Ad-hoc scheduling
+
+In addition to automatic execution, you can create checks to be scheduled manually using the [checks API](../../api/check#the-checkscheckexecute-api-endpoint).
+To create a check with ad-hoc scheduling, set the `publish` attribute to `false` in addition to an `interval` or `cron` schedule.
+
+**Example ad-hoc check**
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "ad_hoc_check",
+    "namespace": "default"
+  },
+  "spec": {
+    "command": "check-cpu.sh -w 75 -c 90",
+    "subscriptions": ["system"],
+    "handlers": ["slack"],
+    "interval": 60,
+    "publish": false
+  }
+}
+{{< /highlight >}}
+
+## Proxy checks {#proxy-requests}
+
+Sensu supports running proxy checks where the results are considered to be for an
+entity that isn’t actually the one executing the check, regardless of whether
+that entity is a Sensu agent entity or a proxy entity.
+Proxy entities allow Sensu to monitor external resources
+on systems or devices where a Sensu agent cannot be installed, like a
+network switch or a website.
+You can create a proxy check using the [`proxy_entity_name` attribute](#using-a-proxy-check-to-monitor-a-proxy-entity) or the [`proxy_requests` attributes](#using-a-proxy-check-to-monitor-a-multiple-proxy-entities).
+
+### Using a proxy check to monitor a proxy entity
+
+When executing checks that include a `proxy_entity_name`, Sensu agents report the resulting events under the specified proxy entity instead of the agent entity.
+If the proxy entity doesn't exist, Sensu creates the proxy entity when the event is received by the backend.
+
+**Example proxy check using a `proxy_entity_name`**
+
+The following proxy check runs every 60 seconds on agents with the `proxy` subscription for the proxy entity `sensu-site`.
```

Comment:
I think we should steer people away from using proxy checks without round robin - using proxy checks without round robin is a working and defined execution mode, but leads to duplicate results. It's generally not what people are after.